app/services/DocumentsService.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import connectors.ReferenceDataConnector
+import models.SelectableList
+import models.reference.DocumentType
+import uk.gov.hmrc.http.HeaderCarrier
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class DocumentsService @Inject() (referenceDataConnector: ReferenceDataConnector)(implicit ec: ExecutionContext) {
+
+  def getDocuments()(implicit hc: HeaderCarrier): Future[SelectableList[DocumentType]] =
+    for {
+      transportDocuments  <- referenceDataConnector.getTransportDocuments().map(_.toSeq)
+      supportingDocuments <- referenceDataConnector.getSupportingDocuments().map(_.toSeq)
+      documents = transportDocuments ++ supportingDocuments
```

Comment:
Rather than doing .map(_.toSeq) for both, you could concatenate the NonEmptySets which will then sort the overall list. This can then be converted to a Seq